tests/fixtures/simulations.py

```diff
@@ -0,0 +1,30 @@
+import functools
+
+import pytest
+
+from openfisca_core.simulation_builder import SimulationBuilder
+
+
+@pytest.fixture
+def simulation_builder():
```

Comment:
I don't have much experience with Simulations, so this might not be a relevant question - but...

Why do we need to create a fixture for `simulation_builder`? It seems to me that `simulation_builder` just returns an newly instantiated "SimulationBuilder" object. Why not instantiate the "SimulationBuilder" object directly in the test, and avoid the confusion of having a fixture?

Reply:
I've just copied it over, but it makes sense, yes.

Reply:
Changed.